issues-policy.md

```diff
@@ -0,0 +1,84 @@
+- **duplicates**: Duplicates are closed with a comment referencing the existing issue.
+- **doc-ok**: The customer is incorrect, and the doc is correct.
+- **forum**: Issues that are support or forum requests are directed to Stack Overflow, or other support sites, and closed.
+
+### Actions on product issues
+
+Depending on the nature of the product issue, we may choose to:
+
+- Transfer the issue to the appropriate product repository.
+- Close the issue as a duplicate of existing product requests.
+- Close the issue with a recommendation to open on the product repository.
+
+Evaluating the correct course of action is subjective. The team members use their judgment on the correct action.
+
+### Actions on content issues
+
+For other issues, the team:
+
+- assigns a priority
+- assigns a milestone, usually "Backlog"
+- reflects tier level support
+- assesses if an issue is a good "up for grabs" issue.
+
+Priority levels are based on the following guidelines, but are subjective. The milestones are also subjective, and will be based on other priorities such as current product release schedules and upcoming launches.
+
+- **P0**: A docs omission or error prevents customers from succeeding in a common scenario.
+  -  ***P0** issues are addressed in the current sprint, taking precedence over previously scheduled work.
+- **P1**: A docs omission or error makes a common scenario much harder, or blocks other well-known scenarios.
+  - **P1** issues are scheduled in a timely manner. Often, P1 issues are planned for an upcoming milestone.
+- **P2**: Issues that cause minor inconveniences, or affect a low page view article.
+  - **P2** issues are generally fixed when an article is updated for higher-priority reasons.
+- **P3**: Issues that are requests for edge case scenarios.
+  - **P3** issues are placed in the backlog and are considered for update when articles are updated for higher-priority reasons.
+
+The diagnosis and triage activities are time-boxed to enable the team to make progress on tasks scheduled for each sprint. Each team member spends at most 30 minutes per day in diagnosis and triage. In order to stay current, diagnosis and triage cannot turn into a resolution session. At most, consider if a single sentence or two can answer a question and provide enough insight for a customer PR.
+
+The **up-for-grabs** label is applied when an issue is a good candidate for a community member (possibly the author) to submit a fix. The team member that applies the **up-for-grabs** label has committed to help community members work through the PR creation process. Issues that are "up for grabs" are often added to the [Community Contribution projects](https://github.com/dotnet/docs/projects/35)
+
+> NOTE: We've only recently adopted the preceding convention. The person that added the label refer you to another team member who will help.
```

Comment:
refer you --> may refer you

Reply:
Also, does it matter that we recently adopted this convention?  How about "has committed to help ..." --> "will help or find someone to help ..."